extensions/Postgres/Postgres/PostgresConfig.cs

```diff
@@ -107,6 +107,23 @@ public class PostgresConfig
     /// </summary>
     public List<string> CreateTableSql { get; set; } = [];
 
+    /// <summary>
+    /// Important: when using hybrid search, relevance scores
+    /// are very different from when using just vector search.
+    /// </summary>
+    public bool UseHybridSearch { get; set; } = false;
+
+    /// <summary>
+    /// Defines the dictionary language the make the textual part of the hybrid Search in postgresql
+    /// see:
+    /// </summary>
+    public string TextSearchLanguage { get; set; } = "english";
+
+    /// <summary>
+    /// Reciprocal Ranked Fusion to score results of Hybrid Search
```

Comment:
what is the "K" for? 

"RRF" => "Reciprocal Ranked Fusion"
"RRFK" => ?

pls add a link to the documentation